fendermint/vm/topdown/src/voting.rs

```diff
@@ -288,14 +314,30 @@ where
     /// Call when a new finalized block is added to the ledger, to clear out all preceding blocks.
     ///
     /// After this operation the minimum item in the chain will the new finalized block.
-    pub fn set_finalized(&self, block_height: BlockHeight, block_hash: V) -> Stm<()> {
+    pub fn set_finalized(
+        &self,
+        parent_block_height: BlockHeight,
+        parent_block_hash: V,
+        proposer: Option<&str>,
+        local_block_height: Option<BlockHeight>,
```

Comment:
what does this parameter mean? Like current fendermint block height?

Reply:
Yes exactly. I think it should be the Tendermint block. We can change the name if you find it confusing.